### ISSUE TITLE
Sending photo by email without user intervention (no Intents)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#298]
```
Heelo,

Maybe add an option to send a photo by email without going through a 
third-party application to avoid prompting by the use.

So if the phone is lost or stolen it is possible to receive a picture 
discreetly.

Example:

Sending Emails without User Intervention (no Intents) in Android
http://www.jondev.net/articles/Sending_Emails_without_User_Intervention_(no_Inte
nts)_in_Android
```

Original issue reported on code.google.com by `laksari....@gmail.com` on 26 Sep 2012 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
Also:

Restrict access by password to the setting for the application to avoid changes 
in case of loss or theft of the phone.
```

Original comment by `laksari....@gmail.com` on 26 Sep 2012 at 10:25

**Comment by GoogleCodeExporter** [comment]
```
The "discreetly sending photo with e-mail" looks interesting.

But setting a password for the preferences of GTalkSMS *does not* increase 
security. The service still could be stopped in an manual or automated fashion 
and it would just add an extra layer of annoyance if the user wants to change a 
preference. Also note that GTalkSMS does not aim to be an after-theft tool. 
There exist far better suited, even open-source and free, solutions in the play 
store.
```

Original comment by `fschm...@gmail.com` on 26 Sep 2012 at 1:00

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e7f28cff8529.
```

Original comment by `yako...@gmail.com` on 7 Mar 2015 at 11:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in beta available on market 5.0.1.84
```

Original comment by `Florent....@gmail.com` on 8 Mar 2015 at 6:07
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 1 May 2015 at 5:42
- Changed state: **Deployed**
